src/styles/animation.scss

```diff
@@ -0,0 +1,46 @@
+.item-enter {
+  max-height: 0;
```

Comment:
Consider using more descriptive class names instead of `.item-enter`. For example, `.accordion-item-enter`. 🎨